### PR TITLE
Kids cats

### PR DESCRIPTION
**Pull request by Maquiner**

### CODE REVIEW

**Review by carlosggarcia — CHANGES_REQUESTED** [review]
I think `self.cat_data` should be only given a value inside `get_catalog`. Here, you're overwriting `self.cat_data` each time you call `get_catalog`. In other words, I'd keep `cat_data = self.get_catalog()` instead of `self.cat_data = self.get_catalog()`.

**Comment by Maquiner** [comment]
Ok let me revert that

File: xcell/mappers/mapper_DESY1wl.py [hunk 1]
```diff
@@ -236,8 +236,8 @@ def get_mask(self):
             print('Loading bin{} mask'.format(self.zbin))
             self.mask = hp.read_map(fname_lite)
         else:
-            cat_data = self.get_catalog()
-            self.mask = get_map_from_points(cat_data, self.nside,
+            self.cat_data = self.get_catalog()
```
Comment (carlosggarcia) [inline xcell/mappers/mapper_DESY1wl.py:239]: missing `self.cat_data`!

File: xcell/mappers/mapper_KV450.py [hunk 2]
```diff
@@ -141,12 +147,33 @@ def _remove_multiplicative_bias(self, cat_data, zbin):
...
+        file_name2 += '.fits.gz'
+        read_lite1, fname_lite1 = self._check_lite_exists(file_name1)
+        read_lite2, fname_lite2 = self._check_lite_exists(file_name2)
+        if read_lite1 and read_lite2:
+            print('Loading bin{} signal map'.format(self.zbin))
+            e1 = hp.read_map(fname_lite1)
+            e2 = hp.read_map(fname_lite2)
+            self.maps[mod] = [-e1, e2]
+        else:
+            print('Computing bin{} signal map'.format(self.zbin))
```
Comment (carlosggarcia) [inline xcell/mappers/mapper_KV450.py:176]: You're not writing the maps after computing them

**Review by carlosggarcia — CHANGES_REQUESTED** [review]

File: xcell/mappers/mapper_KV450.py (hunk 2 above)
Comment (Maquiner) [reply xcell/mappers/mapper_KV450.py:176]: Thank u for spotting these. Now it should all be fine.

File: xcell/mappers/mapper_KV450.py [hunk 3]
```diff
@@ -141,12 +147,33 @@ def _remove_multiplicative_bias(self, cat_data, zbin):
         cat_data['bias_corrected_e2'][sel_gals] /= 1 + self.m[zbin]
 
     def _get_gals_or_stars(self, kind='galaxies'):
-        sel = self.cat['SG_FLAG'] == self.sel[kind]
-        return self.cat[sel]
+        self.cat_data = self.get_catalog()
```
Comment (carlosggarcia) [inline xcell/mappers/mapper_KV450.py:150]: Last `self.cat_data`.

**Review by carlosggarcia — CHANGES_REQUESTED** [review]

**Review by carlosggarcia — APPROVED** [review]

**Pull request merged**